src/tools/peptide-search/components/PeptideSearchForm.tsx

```diff
@@ -50,7 +50,8 @@ import sticky from '../../../shared/styles/sticky.module.scss';
 import '../../styles/ToolsForm.scss';
 
 // just because, no actual known limit
-export const PEPTIDE_SEARCH_LIMIT = 100;
+export const PEPTIDE_SEARCH_MINIMUM_LIMIT = 7;
+export const PEPTIDE_SEARCH_MAXIMUM_LIMIT = 100;
```

Comment:
Those are a bit tricky to understand side-by-side as the 7 is the number of aa in 1 sequence, and the 100 is the number of sequences supported at the same time in one search (that's what the "just because, no actual known limit" comment refers to).
I'd try to find different names compared to the ones here to make things clearer

Reply:
Right. PEPTIDE_SEARCH_AA_MINIMUM_LENGTH or PEPTIDE_SEARCH_SEQ_MINIMUM_LENGTH for the former?
PEPTIDE_SEARCH_SEQUENCES_COUNT for the latter?

Reply:
Either sound good to me 👍🏽